examples/cctl/hello-python/readme.md

```diff
@@ -0,0 +1,99 @@
+## Example python Controller
+This example is taken from [metacontroller](https://metacontroller.app/guide/create/) doc. There is an issue in this example that leads to continuous reconciliations. When _'CompositeController'_ is used with the parent resource's status **undefined** as a sub-resource, then the reconciliation gets into an infinite loop. This happens since _'CompositeController'_ updates _'status.observedGeneration'_ with _'metadata.generation'_ as part of its reconciliation.
```

Comment:
I would rephrase this Paragraph. Now this gets blurry as description start with reference and go straight to exposing problem in original example and explaining how to fix it. I would see this as "standalone" example (without the need to go to other page) to understand what example want to show, with additional section emphasizing the necessity of `subresource: status`.

Reply:
@grzesuav I agree. Describing the example first and then an emphasis on status will make it easy for others to follow.

I am also wondering about docs. I feel it's time to have a separate site for metac as well.